bin/consume_queue.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Consumes and prints items from RabbitMQ queues.
+#
+# Note: Run this in the test container which has access to RabbitMQ.
+#
+# Usage: ./bin/consume_queue.py
+
+import logging
+import os
+import sys
+
+
+# Insert build/ directory in sys.path so we can import pika
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(__file__)),
+        'build'
+    )
+)
+
+# Kill logging so we don't have to listen to Pigeon logging
+logging.basicConfig(level=logging.CRITICAL)
+logging.getLogger().disabled = True
+logging.getLogger('pigeon').disabled = True
```

Comment:
So, when you import `pigeon`, it creates a `CONFIG` which gets all whiny in the logs if there's no `PIGEON_AWS_REGION` set. The reason we don't want to set that in a dev environment is that it uses AWS-fu to decrypt the password and vhost. Thus we don't define it and `CONFIG` gets whiny and then it prints mozlog format shit to stdout and the user is all like, "wtf--i just want to consume RabbitMQ and see if it worked."

There are probably ways to fix it, but every way I could see just moved the dead mouse under a different carpet and I'd like the least amount of dirt in the `pigeon.py` carpet since that's the important thing here.